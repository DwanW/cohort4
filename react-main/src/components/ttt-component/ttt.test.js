import React from 'react';
import {shallow} from 'enzyme';
import Ttt from './ttt';

it('tic-tac-toe component test', ()=>{
    let wrapper = shallow(<Ttt />);
    expect(wrapper).toMatchSnapshot();
});