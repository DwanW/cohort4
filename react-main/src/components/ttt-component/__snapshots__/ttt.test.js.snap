// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tic-tac-toe component test 1`] = `ShallowWrapper {}`;
